.gitignore

```diff
@@ -25,3 +25,4 @@ docs/fn_to_op_table
 docs/op_autodoc
 docs/fn_autodoc
 docs/nvidia.ico
+.DS_Store
```

Comment:
This looks like something Mac specific, right? Not sure, if this should be here.